src/jit/morph.cpp

```diff
@@ -7319,6 +7319,19 @@ bool Compiler::fgCanFastTailCall(GenTreeCall* callee)
         ++nCalleeArgs;
         assert(args->OperIsList());
         GenTreePtr argx = args->gtOp.gtOp1;
+#ifdef _TARGET_ARM_
+        unsigned argAlign = 1;
+        codeGen->InferOpSizeAlign(argx, &argAlign);
+
+        argAlign = roundUp(argAlign, TARGET_POINTER_SIZE) / TARGET_POINTER_SIZE;
+
+        // We don't care float register because we will not use fast tailcall
```

Comment:
It seems like it is worth tracking fastTailCall support for callee functions with floating point. Is there an issue to track adding support for this later?

In addition what is the rational for not implementing it now?